pkg/storage/schema.go

```diff
@@ -80,20 +80,20 @@ func (sm *SchemaManager) MigrateSchemaToVersion(ctx context.Context, version uin
 }
 
 // GetSchemaVersion retrieves the current version of database schema
-func (sm *SchemaManager) GetSchemaVersion(ctx context.Context) (uint, error) {
+func (sm *SchemaManager) GetSchemaVersion(ctx context.Context) (uint, bool, error) {
```

Comment:
My only nit is that externally we now call this Schema State but internally we keep the function name GetSchemaVersion.

Reply:
I agree,  we should probably later rename API endpoint and funcs